src/components/Shortcuts/DeleteShortcut/DeleteShortcut.test.tsx

```diff
@@ -64,20 +66,30 @@ describe('<DeleteShortcut>', () => {
     );
   });
 
-  it.each(['{delete}', '{backspace}'])(
-    'should delete an element with %s key',
-    async (key) => {
-      const activeSlide = activeWhiteboardInstance.getSlide('slide-0');
-      activeSlide.setActiveElementId('element-1');
+  it.each([
+    ['a single element', '{delete}', ['element-1']],
+    ['multiple elements', '{delete}', ['element-1', 'element-2']],
+    ['a single element', '{backspace}', ['element-1']],
+    ['multiple elements', '{backspace}', ['element-1', 'element-2']],
```

Comment:
Maybe could split to several cases.

Reply:
As per discussion I want to apply DRY here. Let's see how this is going.